packages/kerosene/src/datetime/isSameOrBeforeDay.ts

```diff
@@ -0,0 +1,14 @@
+import { isSameDay } from "date-fns";
+import isBeforeDay from "./isBeforeDay";
+
+/**
+ * Returns whether the day of `date` is the same or before the day of `dateToCompare`
+ * @param date
```

Comment:
why have the jsdocs as well as the ts typings here?

Reply:
As far as I know TypeScript recommends that when using JSDoc, types should just be specified in code rather than in JSDoc as well.

Reply:
yeah thats what i mean. you have the ts types, why duplicate that in jsdoc?